src/DataTable/BulkActions.jsx

```diff
@@ -0,0 +1,121 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { MoreVert } from '../../icons';
+import {
+  Button, Dropdown, useWindowSize, Icon, breakpoints,
+} from '..';
+
+export const DROPDOWN_BUTTON_TEXT = 'More actions';
+export const SMALL_SCREEN_DROPDOWN_BUTTON_TEXT = 'Actions';
```

Comment:
In FilterStatus you're using default props for the English strings. Is there a reason not to do that here?

Reply:
There's not, but I think let's leave it for i18n work? I'll open a ticket for it.

Reply:
FilterStatus was written after Stank gave me i18n feedback. ;)